model/src/main/java/org/jboss/pnc/rex/model/requests/StartRequest.java

```diff
@@ -34,4 +36,6 @@ public class StartRequest {
     private final String callback;
 
     private final Object payload;
+
+    private final Map<String, Object> taskResults;
```

Comment:
Super good, I've just realized we would probably want to do this also in StopRequest.

Reply:
Will add it! Thanks for the suggestion! :)